backend/core/src/listings/listings.controller.ts

```diff
@@ -57,6 +58,26 @@ export class ListingsController {
     return mapTo(PaginatedListingDto, await this.listingsService.list(queryParams))
   }
 
+  @Get("includeExternal")
+  @ApiExtraModels(ListingFilterParams, ListingsQueryParams)
+  @ApiOperation({
+    summary: "List listings and optionally include external listings",
```

Comment:
should we add notes on how to use this? For instance "optionally include.."- how do we include or omit?

Reply:
omg! this reminds me! I forgot to run the swagger script. Will do